replicator/src/test/scala/com/evolutiongaming/kafka/journal/replicator/TopicReplicatorSpec.scala

```diff
@@ -57,7 +59,7 @@ class TopicReplicatorSpec extends AnyWordSpec with Matchers {
 
       val data = State(records = List(records))
 
-      val (result, _) = topicReplicator.run(data).get
+      val (result, _) = topicReplicator.run(data).unsafeRunSync()
```

Comment:
Do you have plans on improving `unsafeRunSync()` part?

Reply:
I switched to async suite for `ConsumeTopicTest` and `TopicReplicatorSpec`. However, there are yet `EventualCassandraSpec`, `EventualCassandraTest` and `JournalSpec` using `unsafeRunSync()` but they have quite a complex code to be easily switched to return `Future[Assertion]` as scalatest requires. I would suggest leaving them outside of the scope of this PR